app/home/home.component.ts

```diff
@@ -1,30 +1,48 @@
 import {Component} from 'angular2/core';
+import {RouterLink} from 'angular2/router';
 import {SpinnerComponent} from '../shared/spinner/spinner.component';
 import {CmsService} from '../shared/cms/cms.service';
 
 @Component({
-  directives: [SpinnerComponent],
+  directives: [SpinnerComponent, RouterLink],
   selector: 'publish-home',
   styleUrls: ['app/home/home.component.css'],
   template: `
-    <spinner *ngIf="isLoading"></spinner>
-    <div *ngIf="!isLoading">
-      <h1>Home Page</h1>
-      <p *ngIf="userName">You are logged in as {{userName}}</p>
-      <p *ngIf="!userName">You are not logged in. Shame on you.</p>
+    <div class="main">
+      <section>
+        <spinner *ngIf="!accounts"></spinner>
+        <div *ngIf="accounts">
+          <h1>Your Accounts</h1>
+          <div *ngFor="#account of accounts; #i = index">
+            <h2>{{account.name}} <i>{{account.type}}</i></h2>
+            <ul>
+              <li *ngFor="#story of accountStories[i]">
```

Comment:
It ends up making a GET request to `/authorization/account/:id/stories` for each account.  Which I think just comes back with whatever the CMS kaminari default is?  (10?)

This code in particular is just a "proof of concept" - I'm guessing whomever <cough cough> comes up with the actual UI will do a better job with requests/paging.